addons/languages/songhay/pack/dictionary/prebuilt/additionals.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I guess, you don't need this file?

Reply:
Thanks. I updated it and changed some of the words to to Songhay pronunciation.